lrpar/src/lib/ctbuilder.rs

```diff
@@ -315,6 +326,13 @@ where
         self
     }
 
+    /// Sets the rust edition to be used for generated code.  Defaults to the latest edition of
```

Comment:
Double space after the first `.`.

Reply:
I thought I was mirroring the surrounding text style, but it turns out the other double space was added by me,
I removed it too.

fixed in [bb1e970](https://github.com/softdevteam/grmtools/pull/367/commits/bb1e9702e37f638c0d70545a33661bc8aa47dbd9)